src/standalone/Bucket/Bucket.jl

```diff
@@ -75,13 +75,20 @@ is specified as a function
 of latitude and longitude, but is also treated as constant across
 wavelength; surface is this context refers to soil and vegetation.
 """
-struct BulkAlbedoFunction{FT} <: AbstractLandAlbedoModel{FT}
+struct BulkAlbedoFunction{FT, F <: Function} <: AbstractLandAlbedoModel{FT}
     α_snow::FT
-    α_sfc::Function
+    α_sfc::F
+end
+
+function BulkAlbedoFunction{FT}(
```

Comment:
There must be a default constructor that says `BulkAlbedoFunction(α_snow, α_sfc) == BulkAlbedoFunction{typeof(α_snow), typeof(α_sfc)}(α_snow, α_sfc)`? In many instances in the code (in this PR), we have the former. So we might not need this particular constructor too? Im not sure though, it's nice to have the same format for everything `Struct{FT}(args...)`. up to you!

Reply:
`BulkAlbedoFunction(α_snow, α_sfc) == BulkAlbedoFunction{typeof(α_snow), typeof(α_sfc)}(α_snow, α_sfc)` this is precisely the default constructor for every struct!

I agree that it is nice to have the format with `{FT}`, and I added a constructor to have that. However, I also think that it is a good idea to let julia infer the types when possible. 

We can add `{FT}` around to have symmetry and consistency, but they are not really needed for this specific struct. Let me know what you prefer.

Reply:
:D 

ok! let's keep it as is.